sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -38,7 +40,11 @@ import { Readable } from "stream";
 import { defaultFileFilter, uploadLimits } from "../../utilities/upload-utils";
 import { StudentApplicationDTO } from "../application/models/application.model";
 import { Application } from "../../database/entities";
-import { determinePDStatus } from "../../utilities/student-utils";
+import {
+  determinePDStatus,
+  deliveryMethod,
+} from "../../utilities/student-utils";
+import { credentialTypeToDisplay } from "../../utilities/credential-type-utils";
```

Comment:
i think u missed to push this file

Reply:
you mean credentialTypeToDisplay? its already present. i just reused the utility

Reply:
okay 👍